src/test/regress/expected/multi_shard_update_delete.out

```diff
@@ -0,0 +1,236 @@
+--
+-- multi shard update delete
+-- this file is intended to test multi shard update/delete queries
+--
+ALTER SEQUENCE pg_catalog.pg_dist_shardid_seq RESTART 1440000;
+ALTER SEQUENCE pg_catalog.pg_dist_jobid_seq RESTART 1440000;
+SET citus.shard_replication_factor to DEFAULT;
+SET citus.multi_shard_modify_executor to DEFAULT;
+CREATE TABLE users_copy_table (like users_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_distributed_table('users_copy_table', 'user_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+INSERT INTO users_copy_table SELECT * FROM users_table;
+CREATE TABLE events_copy_table (like events_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_distributed_table('events_copy_table', 'user_id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+INSERT INTO events_copy_table SELECT * FROM events_table;
+CREATE TABLE events_reference_copy_table (like events_reference_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_reference_table('events_reference_copy_table');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+INSERT INTO events_reference_copy_table SELECT * FROM events_reference_table;
+CREATE TABLE users_reference_copy_table (like users_reference_table INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES);
+SELECT create_reference_table('users_reference_copy_table');
+ create_reference_table 
+------------------------
+ 
+(1 row)
+
+INSERT INTO users_reference_copy_table SELECT * FROM users_reference_table;
+-- Run multi shard updates and deletes without transaction on hash distributed tables
+SELECT SUM(value_1) FROM users_copy_table;
+   sum   
+---------
+ 4998894
```

Comment:
Is there a way to get some more predictable outputs? I have no way of telling whether the outputs look right.